engine/spec/components/radio_group_spec.rb

```diff
@@ -1,168 +0,0 @@
-  end
-
-  context "when there are no radio buttons" do
-    let(:subject) do
-      render_inline CitizensAdviceComponents::RadioGroup.new(
-        legend: legend.presence,
-        name: name.presence
-      ) do |c|
-        c.radio_buttons(nil)
-      end
-    end
-
-    let(:name) { "radio-buttons" }
-    let(:legend) { "Radio button group field" }
-
-    it "does not render" do
-      expect(subject.css(".cads-form-field")).to_not be_present
-    end
-  end
-
-  context "when there are optional parameters" do
-    let(:subject) do
-      render_inline CitizensAdviceComponents::RadioGroup.new(
-        legend: legend.presence,
-        name: name.presence,
-        options: {
-          error_message: error_message.presence,
-          optional: optional.presence,
-          hint: hint.presence,
-          size: :small,
-          layout: :inline
-        }
-      ) do |c|
-        c.radio_buttons(radios)
-      end
-    end
-
-    let(:name) { "radio-buttons" }
-    let(:legend) { "Radio button group field" }
-    let(:error_message) { "This is the error message" }
-    let(:optional) { true }
-    let(:hint) { "This is the hint text" }
-    let(:layout) { :inline }
-    let(:size) { :small }
-
-    it "renders the error message" do
-      expect(subject.text.strip).to include "This is the error message"
-    end
-
-    it "renders the hint text" do
-      expect(subject.text.strip).to include "This is the hint text"
-    end
-
-    it "marks the field as optional" do
-      expect(subject.text.strip).to include "optional"
-    end
-
-    it "has the correct layout" do
-      expect(subject.css(".cads-radio-group--inline")).to be_present
-    end
-
-    it "has the correct size buttons" do
-      expect(subject.css(".cads-radio-group--small")).to be_present
-    end
-
-    context "when in Cymraeg" do
-      before do
-        I18n.locale = :cy
-      end
-
-      it "renders optional in Welsh" do
-        expect(subject.text).to include "dewisol"
-      end
-    end
-
-    context "when invalid optional parameter is passed" do
-      let(:optional) { "bananas" }
-
-      it "renders a required input" do
-        without_fetch_or_fallback_raises do
-          expect(subject.text.strip).to_not include "optional"
-        end
-      end
-    end
-
-    context "when invalid size parameter is passed" do
-      let(:size) { :bananas }
-
-      it "renders a normal size input" do
-        without_fetch_or_fallback_raises do
-          expect(subject.css(".cads-from-group--small")).to_not be_present
-        end
-      end
-    end
-
-    context "when invalid layout parameter is passed" do
-      let(:layout) { :bananas }
-
-      it "renders the radio buttons in list layout" do
-        without_fetch_or_fallback_raises do
-          expect(subject.css(".cads-from-group--inline")).to_not be_present
-        end
-      end
-    end
-  end
```

Comment:
@davidsauntson FYI I spotted that these two tests were broken. `cads-from-group--inline` is a typo and we ask if it's **not** present so could never fail. Fixed that in the corrected version.